docs/source/apps.rst

```diff
@@ -126,6 +126,14 @@ Applications
 .. autoclass:: TileOnGridd
     :members:
 
+.. automodule:: monai.apps.pathology.transforms.post.array
+.. autoclass:: CalculateInstanceSegmentationMap
+    :members:
+
+.. automodule:: monai.apps.pathology.transforms.post.dictionary
+.. autoclass:: CalculateInstanceSegmentationMapd
+    :members:
```

Comment:
I think the transform names don't match the exact code, same as the imports and test files.

Thanks.

Reply:
Have updated it, thanks!